changelog.md

```diff
@@ -125,6 +125,11 @@
      * S17c: Modified Burin's description of trolls
    * The South Guard:
      * Fixed various typos
+   * Tutorial:
+     * Remove swamp from map for part 2
```

Comment:
Why remove the swamp hexes? Couldn't we just add an event for them? The tutorial is basically an interactive trailer, I think it is good  to have it show off a bit of terrain variety.

Reply:
There are literally just a couple swamp hexes laying around that look pretty ugly anyway. If we wanted to show off terrain variety, it would involve redrawing the map, not having those hexes.